schema.sql

```diff
@@ -1,5 +1,84 @@
 /* Database schema to keep the structure of entire database. */
-
+-- \c vet_clinic
+DROP TABLE IF EXISTS animals;
 CREATE TABLE animals (
-    name varchar(100)
+    id INT PRIMARY KEY NOT NULL,
```

Comment:
[OPTIONAL]
- Awesome use of the `PRIMARY KEY` and `NOT NULL` constraints. 👏

What do you think about adding the `GENERATED ALWAYS AS IDENTITY` constraint to the `id` column? This ensures that sequential integers that cannot be altered are generated each time a record is being created. I hope it makes sense. 😄 
You can go through this [article](https://www.sqltutorial.org/sql-identity/#:~:text=The%20GENERATED%20ALWAYS%20generates%20sequential,integers%20for%20the%20identity%20column.).

Reply:
This is an amazing idea! 😍 I've implemented it 🤓

Reply:
Tried implementing this one but whoops! GETTING A LOT OF ERRORS 🙈